base/refpointer.jl

```diff
@@ -70,3 +70,10 @@ setindex!(b::RefValue, x) = (b.x = x; b)
 setindex!(b::RefArray, x) = (b.x[b.i] = x; b)
 
 ###
+
+function showcompact(io::IO, b::RefValue)
```

Comment:
Arrays already use `showcompact` for printing their elements, so I didn't think it was necessary. But I can change it if you think it's better that way.